docs/DeveloperGuide.md

```diff
@@ -778,25 +778,271 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
+1. Exiting the application
+   
+   1. Prerequisites: The application must be launched.
+   
+   1. Test case: `exit` <br>
+       Expected: The application exits and the window closes itself automatically.
+   
+### Adding a flashcard
+
+1. Adding a flashcard when in main window of application
+
+   1. Prerequisites: Application must be in main window and not in review or quiz mode.
+   
+   1. Test case (specifying compulsory inputs only): `add q/Does software projects often involve workflows? a/Yes` <br>
+       Expected: A new flashcard is added to the end of the list of flashcards. The compulsory inputs will be set to 
+       what was specified in the command, whereas category will be set to `General` by default when not stated in the
+       input. Result display will output the message: 
+       `New flashcard added:  Question: Does software projects often involve workflows?`
```

Comment:
Maybe can specify that all flashcards will be listed.

Reply:
I have added additional details in